hack/generate-yamls.sh

```diff
@@ -62,10 +62,11 @@ RELEASES=(
 readonly RELEASES
 # Flags for all ko commands
 KO_YAML_FLAGS="-P"
+KO_FLAGS="${KO_FLAGS:-}"
 [[ "${KO_DOCKER_REPO}" != gcr.io/* ]] && KO_YAML_FLAGS=""
 readonly KO_YAML_FLAGS="${KO_YAML_FLAGS} ${KO_FLAGS} --platform=all"
 
-if [[ -n "${TAG}" ]]; then
+if [[ -n "${TAG:-}" ]]; then
```

Comment:
what was the issue here @cardil ? just curious

Reply:
When using strict bash setting of `set -Eeuo pipefail` that fails because of `-u`: fail on unset variable being dereferenced.

Bash could know if variable is unset or empty.